profile/readme.md

```diff
@@ -23,15 +23,15 @@ When connecting to a google cloud sql instance, we are required to connect throu
 <details open>
   <summary>MacOS / linux</summary>
   
-  ```
-  ./cloud_sql_proxy -instances=hidden-display-243419:europe-north1:effekt-db=tcp:3306
+  ```sh
```

Comment:
sh only denotes that it's shell for the formatter, right?

Reply:
Yes, that's right. It is only for the markdown formatter